virt/virt-4-9-release-notes.adoc

```diff
@@ -67,6 +67,7 @@ High-performance virtual machine templates are now available for xref:virt-guest
 [id="virt-4-9-networking-new"]
 === Networking
 //CNV-11197 MAC spoof filtering
+* You can now xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#virt-creating-bridge-nad-cli_virt-attaching-multiple-networks[enable or disable MAC spoof filtering] on secondary networks by configuring a Linux bridge network attachment definition in the CLI.
```

Comment:
link(enable or disable MAC spoof filtering) doesn't point on how to enable and disable macspoof in cni. As an end user we might be looking for config.

Reply:
Also NAD in the cli !! I think we can be bit more specific here . How a yaml can do it .

Reply:
@geetikakay We can only point to the topic title and not a specific item within the topic.

Reply:
Also, "in the CLI" means "by using the CLI". The user needs to update/configure a Linux bridge NAD in the CLI to enable MAC spoof check. The wording is consistent with the topic title.

Reply:
@sjhala-ccs I agree..but no where in the link we see way to enable and disable macspoof.the whole page no where talks about macspoof field

Reply:
@geetikakay The Prerequisites and Procedure (YAML and call out number 6) both mention MAC spoof check. It is the content that was added for CNV-11198. You reviewed the PR https://github.com/openshift/openshift-docs/pull/35806 associated with that story.

Reply:
/LGTM